Tests/ImagineEngineTests/CGImage+LoadTests.swift

```diff
@@ -0,0 +1,31 @@
+/**
+ *  Imagine Engine
+ *  Copyright (c) John Sundell 2017
+ *  See LICENSE file for license
+ */
+
+import Foundation
+import XCTest
+@testable import ImagineEngine
+
+class CGImageLoadTests: XCTestCase {
```

Comment:
If you merge the loading logic back into `BundleTextureImageLoader` it seems like `BundleTextureImageLoaderTests` can also verify this.